libs/src/main/scala/gospeak/libs/youtube/domain/ChannelsResponse.scala

```diff
@@ -0,0 +1,65 @@
+package gospeak.libs.youtube.domain
+
+import com.google.api.services.youtube.model.{ChannelAuditDetails, ChannelBrandingSettings, ChannelContentDetails, ChannelConversionPing, ChannelConversionPings, ChannelListResponse, ChannelLocalization, ChannelSnippet, ChannelStatistics, ChannelStatus, ChannelTopicDetails, InvideoPromotion, TokenPagination, Channel => YChannel}
+
+import scala.collection.JavaConverters._
+
+final case class ChannelsResponse(etag: String,
+                                  eventId: Option[String],
+                                  items: List[Channel],
+                                  kind: String,
+                                  nextPageToken: Option[String],
+                                  prevPageToken: Option[String],
+                                  tokenPagination: Option[TokenPagination],
+                                  visitorId: Option[String])
+
+object ChannelsResponse {
+
+  def apply(channelListResponse: ChannelListResponse): ChannelsResponse = {
+    new ChannelsResponse(
+      channelListResponse.getEtag,
+      Option(channelListResponse.getEventId),
+      channelListResponse.getItems.asScala.map(Channel(_)).toList,
+      channelListResponse.getKind,
+      Option(channelListResponse.getNextPageToken),
+      Option(channelListResponse.getPrevPageToken),
+      Option(channelListResponse.getTokenPagination),
+      Option(channelListResponse.getVisitorId)
+    )
+  }
+}
+
+final case class Channel(etag: String,
+                         id: String,
+                         kind: String,
+                         contentDetails: Option[ChannelContentDetails],
```

Comment:
It's better to not leak Java classes in our model, and only put useful infos (can be extended later)

Reply:
I agree. And it will be one of our discussion topics this afternoon ^^